src/PlotDock.cpp

```diff
@@ -119,16 +133,32 @@ void PlotDock::updatePlot(SqliteTableModel* model, BrowseDataTableSettings* sett
             for(int i=0;i<model->columnCount();++i)
             {
                 QVariant::Type columntype = guessDataType(model, i);
-                if(columntype != QVariant::String && columntype != QVariant::Invalid)
+                if(columntype != QVariant::Invalid)
                 {
                     QTreeWidgetItem* columnitem = new QTreeWidgetItem(ui->treePlotColumns);
-                    // maybe i make this more complicated than i should
-                    // but store the model column index in the first 16 bit and the type
-                    // in the other 16 bits
-                    uint itemdata = 0;
-                    itemdata = i << 16;
-                    itemdata |= columntype;
-                    columnitem->setData(PlotColumnField, Qt::UserRole, itemdata);
+
+                    switch (columntype) {
+                    case QVariant::DateTime:
+                        columnitem->setText(PlotColumnType, "Date/Time");
```

Comment:
These should have a ```tr()``` around them as well.

Reply:
Fixed.